lib/markbind/lib/parser.js

```diff
@@ -91,6 +93,12 @@ Parser.prototype._preprocess = function (node, context, config) {
   element.attribs = element.attribs || {};
   element.attribs[ATTRIB_CWF] = path.resolve(context.cwf);
   if (element.name === 'include') {
+    if (_.isEmpty(element.attribs.src)) {
+      const error = new Error(`Empty src attribute in include in: ${element.attribs[ATTRIB_CWF]}`);
+      this._onError(error);
+      const errorElement = cheerio.parseHTML(utils.createErrorElement(error), true)[0];
+      return Object.assign(element, _.pick(errorElement, ['name', 'attribs', 'children']));
+    }
```

Comment:
Since we want to slowly refactor the code base, perhaps we could split it into its own function? Maybe a `Parser#createErrorElement#(element)`